src/app/static/src/scss/partials/summary.scss

```diff
@@ -0,0 +1,100 @@
+.summary {
+  body {
+    margin-top: 20px;
+  }
+
+  .timeline_area {
+    position: relative;
+    z-index: 1;
+  }
+
+  .single-timeline-area {
+    position: relative;
+    z-index: 1;
+    padding-left: 80px;
+  }
+
+  @media only screen and (max-width: 575px) {
+    .single-timeline-area {
+      padding-left: 100px;
+    }
+  }
+
+  .single-timeline-area .timeline-date {
```

Comment:
`timeline` in general and `timeline-date` in particular seems like misnomers, since this is really just an ordering and doesn't define chronology. `workflow-summary-report` or something similar would make more sense. 

Similarly, it might be more explanatory to rename this file to `workflow-summary.scss`